content/docs/howto/web-servers.md

```diff
@@ -66,6 +66,57 @@ Specifying the HTTP Server version through `buildpack.yml` configuration will
 be deprecated in Apache HTTP Server Buildpack v1.0.0. To migrate from using
 `buildpack.yml` please set the `$BP_HTTPD_VERSION` environment variable.
 
+### Use Default httpd.conf
+
+The Apache HTTPD Server Buildpack supports the ability for a user to just
+provide a static application without having a `httpd.conf` and the buildpack
+will now generate a default `httpd.conf`. In order to activate this workflow
+the `BP_WEB_SERVER` environment variable must be set to `httpd`.
+
+{{< code/copyable >}}
+BP_WEB_SERVER=httpd
+{{< /code/copyable >}}
+
+While this will provide a default configuration there are a few modifications
+that can be made to this `httpd.conf` by setting the following environment
+variables and service bindings.
+
+#### Using BP_WEB_SERVER_ROOT
```

Comment:
```suggestion
#### Set the Root Directory for Static Files
```
Not sure if the wording is clear, but my intent is to _describe_ what the env var does, rather than just naming it.